packages/near-fast-auth-signer/src/hooks/useAuthState.ts

```diff
@@ -42,7 +42,7 @@ export const getAuthState = async (email?: string | null): Promise<AuthState> =>
         networkId,
       });
 
-      await window.fastAuthController.setKey(new KeyPairEd25519(accountInfo.publicKey.split(':')[1]));
+      await window.fastAuthController.setKey(new KeyPairEd25519(accountInfo.keyPair.toString().split(':')[1]));
```

Comment:
too many logic in one line, can we fix it so that if accountInfo is not set, we don't run this part of code?
eg
```
if (accountInfo && accountInfo.keyPair) {
  const keyPairParts = accountInfo.keyPair.toString().split(':');
  if (keyPairParts.length > 1) {
    await window.fastAuthController.setKey(new KeyPairEd25519(keyPairParts[1]));
  }
}
```

Reply:
We already check for accountInfo here: https://github.com/near/fast-auth-signer/blob/0d40cd0318793b59ebd93f2373c41fc618a611f9/packages/near-fast-auth-signer/src/hooks/useAuthState.ts#L36

I would prefer to do it in one line, it's just string operation.

Reply:
but above is for checking `accId` but I'm more concerned about `.keyPair` is not set.

Reply:
I'm not seeing why it could break, typescript it's not showing me any error here. 

I will add the ?. for safety. 

But, if the accountInfo exist, the KeyPair exist, no? The KeyPair come from the passkeys, and we assign the correct KeyPair based on which one is the real one.